### PR TITLE
Migrate `CustomDrugEntrySheet` to user view effects

### PR DESCRIPTION
**Pull request by janhavisinghh**
https://app.shortcut.com/simpledotorg/story/6924/migrate-customdrugentrysheet-to-use-view-effects

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntrySheet.kt [hunk 1]
```diff
@@ -104,6 +104,8 @@ class CustomDrugEntrySheet : BaseBottomSheet<
 
   override fun createEffectHandler(viewEffectsConsumer: Consumer<CustomDrugEntryViewEffect>) = effectHandlerFactory.create(this).build()
 
+  override fun viewEffectsHandler() = CustomDrugEntryViewEffectHandler(this)
```
Comment (msasikanth) [inline app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntrySheet.kt:110]: I think there is a typo here, the method name is just `viewEffectHandler`

**Review by msasikanth — CHANGES_REQUESTED** [review]
Looks good overall, left one comment PTAL

File: app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntrySheet.kt (hunk 1 above)
Comment (janhavisinghh) [reply app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntrySheet.kt:110]: 🤔 Umm seems right to me. @msasikanth It's an overridden method from `BaseBottomSheet`. Or, do you mean we want to change the name in `BaseBottomSheet` to `viewEffectHandler`?

Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntrySheet.kt:110]: Ah, interesting. The `BaseScreen` and `BaseBottomSheet` has different function name. Yep, this looks fine.

**Review by msasikanth — APPROVED** [review]

**Pull request merged**